workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/workflow.spec.yaml

```
@@ -0,0 +1,25 @@
+extension: workflow
+products: ["insightconnect"]
+name: Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack
+title: "Find Indicator in DNS Logs with InsightIDR from Slack"
```

Recommend renaming workflow and all references

```suggestion
title: "Hunt for Threat Indicators in InsightIDR DNS logs from Slack"
```